iogt/static/css/state/_state.scss

```diff
@@ -1,25 +1,231 @@
 
-@media only screen and (min-device-width: 320px) {
-
-  .l-indent--deep {
-    padding: 18px 24px;
-  }
-
+//FEATURE MOBILE
+@media only screen and (min-width: 320px) {
+  @import url(https://fonts.googleapis.com/css?family=Lato:400,700,900);
+  
   @import "_s-menu";
   @import "_s-heading";
   @import "_s-profiles";
   @import "_s-languages";
+  @import "_s-comments";
+  @import "_s-section";
 }
 
+//MOBILE
 @media only screen and (min-width: 600px) {
```

Comment:
This media query should be removed, and incorporated into the one above; Please use TUNE-ME as an example of how this should be done.